packages/govuk-frontend/src/govuk/govuk-frontend-component.mjs

```diff
@@ -8,27 +8,43 @@ import { InitError, SupportError } from './errors/index.mjs'
  *
  * @internal
  * @virtual
+ * @template [T=HTMLElement]
```

Comment:
**question** That's a new syntax to me. Is there a difference with `@template {HTMLElement} T`?

**suggestion** Could we name the `T` as `RootElementType` as it needs to be refered in other places than the same block of documentation?

Reply:
Ah cool, thanks for the explanation, wasn't aware you could give a default without constraining the type. We may need to constrain the type, though, as we call `setAttribute` in the constructor, which requires the type at least to offer that (so be an `Element` at least)?